website/utils/snippets.py

```diff
@@ -178,3 +180,20 @@ def overlaps(check, others, can_equal=True):
             return True
 
     return False
+
+
+def _send_email(to: str, subject: str, body_template: str, context: dict) -> None:
```

Comment:
Because this is now a function that is "exported", so used in multiple places. I think it's better to change it to `send_email` (withouth the leading underscore)

Reply:
I tried so hard, and got so far but in the end it doesn't even matter.
Ik heb het aangepast en nu doet hij het niet meer. Ik heb meerdere van die make tests gedaan en niks maakt het probleem duidelijk.

Reply:
I think your edit was successful tho, the tests are not failing and I see no weird this in the code that suggest it should not work.

Reply:
I took the liberty to rebase the code. To get the latest version locally do:
```
git fetch
git checkout utils_email_send
git reset --hard origin/utils_email_send
git pull
```